src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoReadOnly.cs

```diff
@@ -10,34 +11,26 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoReadOnly
     {
-        // PosTest1: Call ReadOnly on a writable DateTimeFormatInfo instance
-        [Fact]
-        public void TestWritable()
+        public static IEnumerable<object[]> ReadOnly_TestData()
         {
-            DateTimeFormatInfo info = new DateTimeFormatInfo();
-            DateTimeFormatInfo actual = DateTimeFormatInfo.ReadOnly(info);
-
-            Assert.True(actual.IsReadOnly);
+            yield return new object[] { DateTimeFormatInfo.InvariantInfo, true };
+            yield return new object[] { DateTimeFormatInfo.ReadOnly(DateTimeFormatInfo.InvariantInfo), true };
+            yield return new object[] { DateTimeFormatInfo.ReadOnly(new DateTimeFormatInfo()), true };
+            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, false };
+            yield return new object[] { DateTimeFormatInfo.ReadOnly(new CultureInfo("en-US").DateTimeFormat), true };
         }
 
-        // PosTest2: Call ReadOnly on a read only DateTimeFormatInfo instance
-        [Fact]
-        public void TestReadOnly()
+        [Theory]
+        [MemberData(nameof(ReadOnly_TestData))]
+        public void ReadOnly(DateTimeFormatInfo format, bool expected)
```

Comment:
The calls to the method `ReadOnly` have moved inside the MemberData bundle; but before the refactoring the test seemed more about the `ReadOnly` method, and after the `IsReadOnly` property.

I'd expect this test method to be calling ReadOnly.  Admittedly, some of the input may have already called ReadOnly.